examples/self_weight_clamped_truss/self_weight_clamped_truss.jl

```diff
@@ -67,51 +67,79 @@ function structure(N::Int;
     # -------------
     # Mesh
     # -------------
-    mesh = Main.mesh(N, L, A, ϵ_model)
+    s_mesh = create_mesh(N, L, A, ϵ_model)
```

Comment:
Just noting you may use `m` or `msh` as well (both of which seem better than `s_mesh` to me).

Reply:
Done